reference-datasets.yml

```diff
@@ -214,28 +273,28 @@ sources:
   #   metadata:
   #     grid: gx1v7
   #     frequency: annual
-  #     date: 0000
+  #     version: 0000
   #     url: 'https://www.st.nmfs.noaa.gov/copepod/about/databases.html'
   #   args:
   #     urlpath: '/glade/work/kristenk/copepod-biomass/data/POP_gx1v7/*.zarr'
   #     chunks: {}
 
-  tidal-dissipation-jayne-tx0.66v1:
+  tidal-dissipation-jayne-native:
     description: Estimated tidal dissipation data by S.R. Jayne. WHOI.
     driver: netcdf
     metadata:
-      grid: tx0.66v1
+      grid: uniform
```

Comment:
maybe "native" is better than uniform? And we can add text about the "native" grid in the descriptionl

Reply:
Yes, I agree. I will send a PR.